ADOPTERS.md

```diff
@@ -0,0 +1,14 @@
+# Who is using CNOE?
```

Comment:
CNOE or idpbuilder?

Reply:
yeah idpBuilder makes more sense. @fcarta29 can you make the change plz?